### PR TITLE
<Closed>

### PR DESCRIPTION
**Pull request by ghost**
<The pull request has been closed>

### CODE REVIEW

**Comment by diederikdehaas** [comment]
Thanks for this :+1: 

As I'm currently 'redesigning' my LAN and the machines on it, I can't test it soon, but I will once things are back up.

> How can I choose not to merge the README.MD?

With `git reset HEAD~` you can undo your last commit, while keeping the changes. Then do `git add <file1> <file2>` to indicate that you want to commit those. That should leave the README.md modified, but not in the list to be committed. Then you can make your commit.
It's usually best to make changes in a separate branch btw.
So in summary, here's the list of commands (do check them as I'm doing it by heart):

```
git reset HEAD~
git checkout add-vht_capab
git add include/wifi.h os_dep/linux/ioctl_cfg80211.c
git commit -v
<write commit msg in your favorite editor>
git push
```

Then you can probably best do a new PR in which you offer to merge branch `add-vht_capab` into my branch driver-4.3.22-beta

HTH
